testing/toDoListPart2.test.js

```diff
@@ -0,0 +1,127 @@
+import { jest } from '@jest/globals';
+import removeAllCompleted from '../src/modules/removeAll.js';
+import updateCompletedStatus from '../src/modules/checkChange.js';
+import editTaskDescription from '../src/modules/editTask.js';
```

Comment:
- [ ] As required for the project, kindly ensure that all your tests are passing as right now no one of them is passing, they are all failing.
![Screenshot from 2023-06-15 20-05-11](https://github.com/PabloBona/to-Do-List/assets/83042742/0a996210-4b3b-4b09-b8f7-0dad1ac515ce)

Reply:
@tresorsawasawa 
Hi, all my test are passing:
![image](https://github.com/PabloBona/to-Do-List/assets/110569486/492f3908-1ea1-4994-ba90-1fbfc7416765)

Plz check it again

Reply:
I also ask a Micronauts partner to check if the code works:
![image](https://github.com/PabloBona/to-Do-List/assets/110569486/4bc47215-0c86-49d1-b141-bcbed81aab84)
I noticed that you didn't provide any advice or reasons for the required changes. It would be really helpful if you could let me know what specifically needs improvement so that I can make the necessary adjustments. Additionally, all you need to do is run an "npm install" command.
@tresorsawasawa 
The code works

Reply:
Hi @PabloBona and @M-Anwar-Hussaini 

Here is the error am still getting:
![Screenshot from 2023-06-15 21-49-30](https://github.com/PabloBona/to-Do-List/assets/83042742/5e41f10d-5f12-4002-9b78-43cb744cd274)

Reply:
@tresorsawasawa 
Can you tell me how to solve this error, because neither I nor my partner or other colleagues have it. And I don't know how to do it.

Reply:
https://www.loom.com/share/f3bb19e1de0645fb81680656c4e8bf80
I leave you the video of how my code works, I hope it works for you

Reply:
Hi @PabloBona and @M-Anwar-Hussaini 

To fix it, just update the line 8 , in your `package.json` file:
current one: `"test": "set NODE_OPTIONS=--experimental-vm-modules && npx jest --verbose",`
Must be updated to this: `"test": "NODE_OPTIONS=--experimental-vm-modules npx jest --verbose",`